lib/px/pxnginx.lua

```diff
@@ -51,8 +51,31 @@ function M.application(file_name)
     local user_agent = ngx.var.http_user_agent or ""
     local string_sub = string.sub
     local string_len = string.len
+    local cjson = require "cjson"
     local pcall = pcall
 
+    local function send_enforcer_telemetry()
+        -- check if report_active_config exist, and true,
+        -- if nil treat as true, for first time send
+        if px_config.repot_active_config == nil or px_config.repot_active_config then
```

Comment:
should be `report_active_config` (you're missing an r)

Reply:
👍